controllers/quizController.js

```diff
@@ -0,0 +1,59 @@
+import * as QuizModel from '../models/QuizModel';
+import AppError from '../errors/AppError';
+
+const getQuizzes = async (req, res, next) => {
+  try {
+    const quizzes = await QuizModel.getAllQuizzes();
+    res.status(200).send({
+      payload: { quizzes }
+    })
+  } catch (error) {
+    next(new AppError(error.message));
+  }
+}
+
+const addQuiz = async (req, res, next) => {
+  try {
+    const quiz = await QuizModel.save({
+      ownerId: req.body.userId,
+      name: req.body.name,
+      description: req.body.description
```

Comment:
Questions are missing, but we need to check how quiz saving will be handled by front end